packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -898,12 +926,28 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// but doesn't currently have anything to show in the menu.
   final bool enabled;
 
+  /// If provided, the shape used for the menu.
+  ///
+  /// If this property is null, then [PopupMenuThemeData.shape] is used.
+  /// If [PopupMenuThemeData.shape] is also null, then the default shape for
+  /// [MaterialType.card] is used. This default shape is a rectangle with
+  /// rounded edges of BorderRadius.circular(2.0).
+  final ShapeBorder shape;
+
+  /// If provided, the background color used for the menu.
+  ///
+  /// If this property is null, then [PopupMenuThemeData.color] is used.
+  /// If [PopupMenuThemeData.color] is also null, then
+  /// [Theme.of(context).cardColor] is used.
```

Comment:
nit, since this wouldn't create the link you're expecting: 
```suggestion
  /// Theme.of(context).cardColor is used.
```

Reply:
Done